testscommon/subRounds/subRoundStartExtraSignersHolderMock.go

```diff
@@ -16,8 +16,8 @@ func (mock *SubRoundStartExtraSignersHolderMock) Reset(pubKeys []string) error {
 	return nil
 }
 
-// RegisterExtraSingingHandler -
-func (mock *SubRoundStartExtraSignersHolderMock) RegisterExtraSingingHandler(extraSigner consensus.SubRoundStartExtraSignatureHandler) error {
+// RegisterExtraSigningHandler -
+func (mock *SubRoundStartExtraSignersHolderMock) RegisterExtraSigningHandler(extraSigner consensus.SubRoundStartExtraSignatureHandler) error {
```

Comment:
there still are some `Singing` instead of `Signing` - found 8 occurences

Reply:
Right, changed all remaining ones